contracts/Vault.sol

```diff
@@ -102,10 +103,11 @@ contract Vault {
     ) internal {
         mapMemberSynth_lastTime[_member][_synth] = block.timestamp; // Time of deposit
         mapMemberSynth_deposit[_member][_synth] += _amount; // Record deposit
-        uint256 _weight = iUTILS(UTILS()).calcValueInBase(iSYNTH(_synth).TOKEN(), _amount);
+        uint256 _weight = iUTILS(UTILS()).calcSwapValueInBase(iSYNTH(_synth).TOKEN(), _amount);
```

Comment:
is this sufficient to close #100 ? Or is this require also necessary:

require(isCurated(iSYNTH(_synth).TOKEN()))